frontend/src/scenes/experiments/ExperimentMetricForm.tsx

```diff
@@ -165,9 +175,10 @@ export function ExperimentMetricForm({
                 </LemonLabel>
             </div>
             {/* :KLUDGE: Query chart type is inferred from the initial state, so need to render Trends and Funnels separately */}
-            {metric.metric_type === ExperimentMetricType.MEAN &&
-                metric.source.kind !== NodeKind.ExperimentDataWarehouseNode && <Query query={queryConfig} readOnly />}
-            {metric.metric_type === ExperimentMetricType.FUNNEL && <Query query={queryConfig} readOnly />}
+            {isExperimentMeanMetric(metric) && metric.source.kind !== NodeKind.ExperimentDataWarehouseNode && (
+                <Query query={queryConfig} readOnly />
+            )}
+            {isExperimentFunnelMetric(metric) && <Query query={queryConfig} readOnly />}
```

Comment:
We are always using the same `Query` component. If type safety allows, we can do:

```ts
{metric.source.kind !== NodeKind.ExperimentDataWarehouseNode && (
    <Query query={queryConfig} readOnly />
)}
```

But I'm not sure if we need the type guards to narrow `queryConfig`.